BookingsApi/BookingsApi.DAL/Commands/AddParticipantsToVideoHearingCommand.cs

```diff
@@ -62,9 +63,9 @@ public async Task Handle(AddParticipantsToVideoHearingCommand command)
 
             _context.Update(hearing);
             
-            var participants = await _hearingService.AddParticipantToService(hearing, command.Participants);
+            await _hearingService.AddParticipantToService(hearing, command.Participants);
 
-            await _hearingService.CreateParticipantLinks(participants, command.LinkedParticipants);
+            await _hearingService.CreateParticipantLinks(hearing.Participants.ToList(), command.LinkedParticipants);
```

Comment:
could we add test coverage for this? Maybe something that asserts the participant links have been created

Reply:
Added. If you could re-review that would be great!